doc/source/devref/quality_of_service.rst

```diff
@@ -2,52 +2,54 @@
 Quality of Service
 ==================
 
-Quality of Service advanced service is designed as a service plugin. The
-service is decoupled from the rest of Neutron code on multiple levels (see
+The Quality of Service (QoS) advanced service is designed as a service plugin. This
+service is decoupled from the rest of neutron code on multiple levels (see
```

Comment:
I looked that up in the OpenStack glossary and only saw it lower-cased.  It was capitalized inconsistently in this doc, which is why I thought to deal with this...